hax/treemakers/corrections.py

```diff
@@ -215,7 +290,7 @@ def extract_data(self, event):
         )
         result['cs1_tpf_2dfdc'] = s1.area * result['s1_xyz_correction_tpf_fdc_2d']
 
-        cvals = [result['x_3d_nn'], result['y_3d_nn'], result['z_3d_nn']]
+        cvals = [result['x_3d_nn_tf'], result['y_3d_nn_tf'], result['z_3d_nn_tf']]
         result['s1_xyz_correction_nn_fdc_3d'] = (
```

Comment:
Should we rename this variable so it's not confused with the previous one? Or perhaps make another class like the ```ReconstructedPosition``` class in pax, so we can store this quantity for each algorithm? (In case we still want to do comparisons.)

(```s1_area_fraction_top*``` variables below should also fit into this class...)

Reply:
I don't see the need to add these variables in PatternReconstruction (name changed as suggested), it can be done of course. I think for PostSR1 analysis, people are mostly using the same data - PAX_v6.8.0. For high-E analysis, we probably won't use these variables for analysis